docs/source/models/cause.rst

```diff
@@ -337,14 +336,38 @@ Mortality Impacts
 Cause-Specific Mortality
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+The cause-specific mortality rate is the mortality rate from a specified cause for a population. 
+The numerator is the number of deaths attributed to a specific cause. The denominator remains the 
+size of the population. In GBD, cause-specific estimates by country, year, age, sex.
```

Comment:
denominator is person-years, not persons---that is what makes it a rate

Reply:
to be consistent with how we've been phrasing this in the incidence section, you could say that the denominator is "person-time in the total population"